stock_shipment_schedule_report/wizards/shipment_schedule_report_wizard.py

```diff
@@ -21,12 +21,18 @@ class ShipmentScheduleReportWizard(models.TransientModel):
              "for customers. If unselected, consider all the internal "
              "locations",
     )
+    website_published = fields.Boolean(
+        string='Published On Website',
```

Comment:
It could mean "it will only show products that are sold on website", is this fine?
May be "Products Published on Website" or "Products Available on Website" would be better?